common/misc.go

```diff
@@ -19,10 +27,18 @@ func HardResetHNS() error {
 	log.Debugln("Removing HNS program data")
```

Comment:
I don't think this is possible. If service missbehaves, I think Windows kills it?